docs/tutorial/kubernetes-charm-django.rst

```diff
@@ -544,7 +544,7 @@ In this iteration, we'll add a greeting app that returns a ``Hello, world!`` gre
 The generated Django app doesn't come with an app, which is why
 we had to initially enable debug mode for testing.  We will need to go back
 out to the ``~/django-hello-world`` directory where the rock is and enter
-into the ``/django_hello_world`` directory where the Django app
+into the ``./django_hello_world`` directory where the Django app
```

Comment:
FWIW, for me, relative vs. absolute pathing doesn't aid in understanding the folder structure. My confusion comes from having two dirs with the same name differentiated by punctuation, one nested in the other.